pkg/scheduler/objects/application.go

```diff
@@ -137,6 +147,37 @@ func NewApplication(siApp *si.AddApplicationRequest, ugi security.UserGroup, eve
 	return app
 }
 
+// Adjust the placeholder timeout if "creationTime" is defined. It's used during Yunikorn restart,
+// in order to properly track how long a placeholder pod should be in "Running" state.
+func (sa *Application) adjustPlaceholderTimeout(placeholderTimeout time.Duration, siApp *si.AddApplicationRequest) (bool, time.Duration) {
+	creationTimeTag := siApp.Tags["creationTime"]
+	var creationTime int64
+	if creationTimeTag != "" {
+		var err error
+		creationTime, err = strconv.ParseInt(creationTimeTag, 10, 64)
+		if err != nil {
```

Comment:
Move to `common.ConvertSITimestamp(ts string) time.Time` for the conversion.

Does this work as a simpler option to do the calculation:
```
created := common.ConvertSITimestamp(creationTimeTag)
newPT := created.Add(placeholderTimeout).sub(time.Now())
```